capstone/src/main/resources/static/script.js

```diff
@@ -0,0 +1,76 @@
+
+/** checks whether the user is authenticated and adjust elements 
+according to whether the user is logged in or logged out */
+function checkAuth(){
+  // send request for information on login status
+  fetch('_gcp_iap/identity').then(response => response.json()).then((data) => {
+    const userEmail = document.getElementById('email-form');
+    // default value set for testing on dev server
+    const email = "accounts.google.com:jennysheng@google.com";
```

Comment:
I understand you want to be able to run locally. So, is it possible to detect that data is empty, and only then hard-code the email?